content/en/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -1,184 +0,0 @@
----
-title: Garbage Collection
-content_type: concept
-weight: 60
----
-
-<!-- overview -->
-
-The role of the Kubernetes garbage collector is to delete certain objects
-that once had an owner, but no longer have an owner.
-
-
-<!-- body -->
-
-## Owners and dependents
-
-Some Kubernetes objects are owners of other objects. For example, a ReplicaSet
-is the owner of a set of Pods. The owned objects are called *dependents* of the
-owner object. Every dependent object has a `metadata.ownerReferences` field that
-points to the owning object.
-
-Sometimes, Kubernetes sets the value of `ownerReference` automatically. For
-example, when you create a ReplicaSet, Kubernetes automatically sets the
-`ownerReference` field of each Pod in the ReplicaSet. In 1.8, Kubernetes
-automatically sets the value of `ownerReference` for objects created or adopted
-by ReplicationController, ReplicaSet, StatefulSet, DaemonSet, Deployment, Job
-and CronJob.
-
-You can also specify relationships between owners and dependents by manually
-setting the `ownerReference` field.
-
-Here's a configuration file for a ReplicaSet that has three Pods:
-
-{{< codenew file="controllers/replicaset.yaml" >}}
-
-If you create the ReplicaSet and then view the Pod metadata, you can see
-OwnerReferences field:
-
-```shell
-kubectl apply -f https://k8s.io/examples/controllers/replicaset.yaml
-kubectl get pods --output=yaml
-```
-
-The output shows that the Pod owner is a ReplicaSet named `my-repset`:
-
-```yaml
-apiVersion: v1
-kind: Pod
-metadata:
-  ...
-  ownerReferences:
-  - apiVersion: apps/v1
-    controller: true
-    blockOwnerDeletion: true
-    kind: ReplicaSet
-    name: my-repset
-    uid: d9607e19-f88f-11e6-a518-42010a800195
-  ...
-```
-
-{{< note >}}
-Cross-namespace owner references are disallowed by design.
```

Comment:
This content appears to have been lost in this rework and is critical to communicate

Reply:
Opened https://github.com/kubernetes/website/issues/29480 to track restoring that bit